bonfire/resources/ephemeral-clowdenvironment.yaml

```diff
@@ -22,17 +22,16 @@ objects:
         port: 8000
         mode: operator
       featureFlags:
-        mode: none
+        mode: local
       metrics:
         port: 9000
-        mode: operator
         path: "/metrics"
+        prometheus:
+          deploy: true
+        mode: operator
       kafka:
-        namespace: default
-        clusterName: my-cluster
-        mode: local
+        mode: operator
```

Comment:
Not quite ready for this yet till the Clowder status PR goes in ;)

Reply:
It's merged now :tada:

Reply:
Testing

Reply:
Somehow the ClowdEnvironment status still went to 'ready' before a KafkaConnect pod was up .... debugging it.

Reply:
```
❯ oc get deployment -n test
NAME                       READY     UP-TO-DATE   AVAILABLE   AGE
env-test-connect-connect   1/1       1            1           3m6s
env-test-entity-operator   1/1       1            1           4m26s
env-test-featureflags      1/1       1            1           6m32s
env-test-minio             1/1       1            1           6m33s
featureflags-db            1/1       1            1           6m32s
```

And on the ClowdEnvironment:
```
  status:
    deployments:
      managedDeployments: 3
      readyDeployments: 3
    generation: 2
    ready: true
    targetNamespace: test
```

Something is not adding up here.

Reply:
OK, feature flag was not enabled on minikube. Working there now. Will test on ephemeral cluster next.